### ISSUE TITLE
grepwin misses files and drops files in subsequent searches

### ISSUE THREAD

**Issue by smskt** [stefankueng/grepWin#416]
Latest version of GrepWin 2.0.13 appears to miss files with matching content.
Searching a folder in C:\ProgramData for *.log files that contan a simple string often fails to find file that contain the string.

I see the 1st search find one file when there are more than one that match the criteria
However executing a 2nd/3rd/4th search the search always comes back empty, it does not even find the first file.

In the example below all 25 files contain the TRACE string, only one was identified with a match.
Also happens when running grepwin as Administrator.
Files are a few MB large.
Older version of grepwin 1.9.2 finds all files.

![GrepWinConfig](https://github.com/stefankueng/tools/assets/107920394/d8e57bc7-75fe-4890-a386-5505afaf3365)

**Comment by stefankueng** [comment]
could it be that the log files get bigger than 2000kb? because that's the limit you set for the size.

**Comment by smskt** [comment]
Valid point, it shows that in the screenshot but I did select all sizes as well and it had the same issue.
You are correct though most of the log file sizes are around 20MB so they are quite large.

The odd thing was that it would find one file initially but in a subsequent search didn't find that file either. 
Not sure what to make of it.

I installed v1.9 yesterday to get a working version on my system.
However, to retest the All Sizes option after your comments I just reinstalled the latest v2 again and find it currently finds those files.
I had this issue on and off for a while on separate systems and I suspect this must be a random thing.
I will observe and follow up if I have this issue again.
